README.md

```diff
@@ -32,6 +32,10 @@ Clone the repository
 
   - Production: `$ fvm flutter run --flavor production`
 
+- Generate assets folder
+  
+  - `$ fvm flutter packages pub run build_runner build`
```

Comment:
also add ` --delete-conflicting-outputs` just in case 🙆‍♂️  we can delete all the current generated files that we might encounter when comparing between the changes from remote vs local changes.

Reply:
@sleepylee Good to know